lib/matplotlib/tests/test_axes.py

```diff
@@ -4603,6 +4603,36 @@ def test_hist_stacked_bar():
     ax.legend(loc='upper right', bbox_to_anchor=(1.0, 1.0), ncols=1)
 
 
+@pytest.mark.parametrize("histtype", ["step", "stepfilled"])
+@pytest.mark.parametrize("color", [["blue", "green", "brown"], [None]*3])
```

Comment:
The current modification handles [None, "blue", None] by assigned the rcparam color to None and uses the specified color wherever specified. Should be fine?

Reply:
Default param seems more logical to me

Reply:
I'm not aware that we resolve `None` in parameter sequences. I've explicitly checked `stackplot()`, which is conceptually similar in that the color goes over multiple datasets and if not given, colors are taken from the default color cycle. Thus, I suggest to error out in case of `None` elements in the sequence (maybe not explicitly handling in l.7017 is enough).